File: WordApp/ClientApp/src/app/models/users/user-login.model.ts
import {EUserType} from './../../app-enums';

export class UserLoginModel {
  public login: string;
  public password: string;
  public token: string;
  public tokenExpirationTime: Date;
  public userType: EUserType;
  public name: string;
  public id: string;
  constructor(login: string, password: string, token: string, tokenExpirationDate: Date) {
    this.login = login;
    this.password = password;
    this.token = token;
    this.tokenExpirationTime = tokenExpirationDate;
  }
}

File: WordApp/Models/User/UserLoginModel.cs
using System;
using Entities.Enums;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;
using WordApp.Models.Base;

namespace WordApp.Models.User
{
    public class UserLoginModel: BaseModel
    {
        public string Login { get; set; }
        public string Password { get; set; }
        public string Token { get; set; }
        public DateTime? TokenExpirationTime { get; set; }
        [JsonConverter(typeof(StringEnumConverter))]
        public UserType UserType { get; set; }
        public string Name { get; set; }
    }
}
